action.yml

```diff
@@ -21,6 +21,12 @@ inputs:
   suffix:
     description: Additional suffix to append to every branch name
     required: false
+  dry_run:
```

Comment:
I don't think there is any need for this input. Why it was added?

Reply:
This allows users of the action to see what branches it will delete without deleting it. This is meant to operate like most dry-run command line flags.